### PR TITLE
[RFC] Fix classloading error when using contao/phpstan in contao based projects

### PR DESCRIPTION
**Pull request by bytehead**
This could be a possible fix for https://github.com/contao/contao/issues/627.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
     },
     "autoload": {
         "psr-4": {
-            "Contao\\PhpStan\\": "src/"
+            "Contao\\PhpStan\\": "src/",
+            "Contao\\CoreBundle\\Tests\\": "../../contao/core-bundle/tests/"
```
Comment (leofeyer) [inline composer.json:44]: Do we not have to remove the entry from `autoload-dev` then?

**Comment by bytehead** [comment]
I think we need `autoload-dev` for `contao/phpstan` itself in test environments.
My added line only will provide autoload information if you use `contao/phpstan` in your project. 
If you remove it, the tests for `contao/phpstan` will fail with the same error message as in contao/contao#627.

File: composer.json (hunk 1 above)
Comment (bytehead) [reply composer.json:44]: I think we need `autoload-dev` for `contao/phpstan` itself in test environments.
My added line only will provide autoload information if you use `contao/phpstan` in your project. 
If you remove it, the tests for `contao/phpstan` will fail with the same error message as in contao/contao#627.

Comment (leofeyer) [reply composer.json:44]: I see. Then I guess the solution is fine. @aschempp WDYT?

Comment (aschempp) [reply composer.json:44]: Will it work with `contao/contao` and `contao/core-bundle` installed?

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**